src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -46,10 +46,12 @@ fun testFailed(
     exceptions: Array<String>
 ) {}
 
+internal val ideaPluginEnabled = ideaPluginEnabled()
```

Comment:
Please write a comment that by having this property, we limit the number of `ideaPluginEnabled` calls to 1. So, at the plugin side, the callback is called only once.